### DISCUSSION TITLE
L10n/i18n

### DISCUSSION THREAD

**Discussion by lsap** [Bogdan17-bit/Trading-Platform#3] [Ideas]
Sorry for adding the idea of more flexible translations to issues as it's rather an idea in future.

It's not an ad, but your project could have desktop (might be Web as well later) version of online translators [integration](https://github.com/crow-translate/crow-translate) .

**Comment by Bohdan17-bit** [comment]
Thank you!
I will keep your comments in mind and check out this repo.
Looks very cool!
